styles/styles.css

```diff
@@ -795,6 +802,61 @@ main .section.responsive-title h1 {
 }
 
 /* REDESIGN GLOBAL STYLES */
-:root.redesign-v2 {
+.redesign-v2 {
+  --v2-space-small: 16px;
+
   font-size: 16px;
-}
\ No newline at end of file
+}
+
+/* stylelint-disable-next-line no-descending-specificity */
+.redesign-v2 .section {
+  padding: 40px 0;
+}
+
+/* stylelint-disable-next-line no-descending-specificity */
+.redesign-v2 .section > div {
+  padding: 0 var(--v2-space-small);
+  margin: 0 auto;
+}
+
+.redesign-v2 .section > .full-width {
+  padding: 0;
+  margin: 0;
+  width: 100%;
+  max-width: 100%;
+}
+
+.redesign-v2 main .section.padding-0 {
+  padding: 0;
+}
+
+.redesign-v2 .with-marker::before {
+  content: '';
+  margin: 0;
+  height: 0;
+  width: 45px;
+  border-bottom: 4px solid var(--c-accent-red);
+  display: block;
+  position: relative;
+  top: -18px;
+}
+
+@media (min-width: 1040px) {
```

Comment:
Not sure what is defined in Mack, but shouldn't this be 1200px? Or we are not using Volvo breakpoints? If that's the case, I would like to know which is the decision on breakpoints for Mack :)

Reply:
I moved the code for header.js (v1). Because the header height was set in the header.js there was a layout shift (the hero banner was pushed by the expanding header when the `header.css` was fetched).
So the code is from the header v1 and for the header v1.
In the scope of the header v2 we will need to update the code. It is possible that the heading height is different in the new design, but this is out of the scope for now